style.css

```diff
@@ -326,14 +581,63 @@ li {
   }
 }
 
-@media screen and (max-width: 480px) {
-  #logo {
-    width: 100px !important;
+/* couldn't find any bootstrapp class for this  */
+
+@media screen and (max-width: 768px) {
+  .p-stories {
+    margin-top: 80px;
   }
 }
 
-@media screen and (max-width: 380px) {
+/* couldn't find any bootstrapp class for this  */
+
+@media screen and (max-width: 480px) {
+  .my-turn-img {
+    height: 246px;
```

Comment:
Same with 480px . Already used in line no. 394. Use only one media query for one screen.